libs/crypto/src/x509_cert.rs

```diff
@@ -0,0 +1,19 @@
+use crate::Sha256Digest;
+
+pub use ::x509_cert::der;
+pub use ::x509_cert::der::pem;
+pub use ::x509_cert::Certificate;
+
+use ::sha2::{Digest, Sha256};
+
+pub fn x509_public_key_s256(certificate: &Certificate) -> Option<Sha256Digest> {
```

Comment:
Can we make it a TryFrom<&Certificate> for Sha256Digest or something like that? :D